yamlReader.py

```diff
@@ -0,0 +1,88 @@
+import yaml
+import start
+
+# yaml file opening and loading
+yamlName = start.yamlName()
+yamlData = yaml.safe_load(open(yamlName))
+
+
+def getDefault():
+    defaultDay = None
+
+    for (attribute, value) in yamlData.items():
+        # fetching employees list
+        if (attribute == "default_date_day"):
+                defaultDay = int(value)
+    return defaultDay
+
+
+def getEmployeesList():
```

Comment:
This is bullshit, this function should be one liner:
```
def getEmployeesList():
    return yamlData.get('employees')
```
Same goes for some other functions as well.

Reply:
Debullshitified enough? :)